src/Telemetry/AzureDevOpsClient/AzureDevOpsClientFactory.cs

```diff
@@ -10,9 +11,20 @@ public sealed class AzureDevOpsClientFactory : IAzureDevOpsClientFactory
     {
         public AzureDevOpsClientFactory() { }
 
-        public IAzureDevOpsClient CreateAzureDevOpsClient(string baseUrl, string organization, int maxParallelRequests, string accessToken)
+        public IAzureDevOpsClient CreateAzureDevOpsClient(
+            string baseUrl,
+            string organization,
+            int maxParallelRequests,
+            string accessToken,
+            IHttpClientFactory clientFactory)
```

Comment:
Instead of taking this in as a parameter in the Create method, can we dependency inject it instead?

Reply:
I had a look at the code, I don't think it would be that easy to do this for a number of reasons, and would further increase the scope of this PR. Maybe we could do this in a follow up PR?

Reply:
You've already merged this, however, if you plan to do this work in the future, you should write an issue to follow up. Also, having been on projects that did DI later, it was a lot more work than it would've been if it were done up front.

Reply:
Here's the followup issue: https://github.com/dotnet/arcade/issues/9665